templates/book/book_template.html

```diff
@@ -1,230 +1,184 @@
-    <div class="top">
-    </div>
-    <div class="content">
-      <div class="zosia_title">
-        Zimowy Obóz Studentów Informatyki A
-      </div>
-      <div class="zosia_logo"> <img src="../assets/zosia.png"> </div>
-      <div class="date_and_place">
-        Przesieka <br>
-        {{ camp_date }}
-      </div>
-      <div class="logo_wrapper">
-        <div class="uwr_logo"> <img src="../assets/logo_UWr.png"> </div>
-        <div class="ksi_logo"> <img src="../assets/logo_KSI.svg"> </div>
+  <head>
+    <meta charset="utf-8" />
+    <link href="book.css" rel="stylesheet" />
+    <title>Book</title>
+    <meta name="description" content="Book" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <div class="title_page">
+      <div class="top"></div>
+      <div class="content">
+        <div class="zosia_title">Zimowy Obóz Studentów Informatyki A</div>
+        <div class="zosia_logo"><img src="../../assets/zosia.png" /></div>
+        <div class="date_and_place">
+          {{ place.localization }} <br />
+          {{ camp_date }}
+        </div>
+        <div class="logo_wrapper">
+          <div class="uwr_logo"><img src="../../assets/logo_UWr.png" /></div>
+          <div class="ksi_logo"><img src="../../assets/logo_KSI.svg" /></div>
+        </div>
       </div>
+      <div class="bottom"></div>
     </div>
-    <div class="bottom">
-    </div>
-  </div>
 
-  <div class="info_page">
-    <div class="contact">
-      <b> Kontakt z organizatorami </b> <br />
-      <table>
-        {% for person, phone in contacts.items() %}
-        <tr>
-          <td>{{ person }}</td>
-          <td>{{ phone }}</td>
-        </tr>
-        {% endfor %}
-      </table>
+    <div class="info_page">
+      <div class="contact">
+        <b> Kontakt z organizatorami </b> <br />
+        <table>
+          {% for person, phone in contacts.items() %}
+          <tr>
+            <td>{{ person }}</td>
+            <td>{{ phone }}</td>
+          </tr>
+          {% endfor %}
+        </table>
+      </div>
     </div>
-  </div>
 
-  <div class="plan_pages">
-    {% for d in days %}
-    <div class="event_day">
-      <div class="day_name">
-        {{ d.name }}
+    <div class="plan_pages">
+      {% for d in days %}
+      <div class="event_day">
+        <div class="day_name">{{ d.name }}</div>
+        <table class="event_table">
+          <tbody>
+            {% for e in d.events %}
+            <tr
+              class="event{% if e.highlight != 'none' %} highlight_{{ e.highlight }}{% endif %}"
+            >
+              <td class="hour_container">
+                <div class="hour">{{ e.startTime }} {{ e.endTime }}</div>
```

Comment:
What about adding `-` between start time and end time?

Reply:
It is not needed because `starrtTime` will always be above `endtime`.